lib/elixir/lib/macro.ex

```diff
@@ -315,6 +315,20 @@ defmodule Macro do
   def update_meta(other, _fun) do
     other
   end
+  @doc """
```

Comment:
@josevalim do we plan on exposing/documenting this, or should we start with internal usage only and `@doc false`?